js/output/pjs/pjs-code-injector.js

```diff
@@ -156,13 +156,46 @@ class PJSCodeInjector {
 
             playSound: (sound) => {
                 if (sound && sound.audio && sound.audio.play) {
+					if( sound.audio.fadeOutInterval ) {
+						sound.audio.fadeOutStopInterval();                                    
+					}
+
                     sound.audio.currentTime = 0;
                     sound.audio.play();
                 } else {
                     throw {message: i18n._("No sound file provided.")};
                 }
             },
 
+			stopSound: (sound, fadeOut) => {
+				if (sound && sound.audio && sound.audio.volume) {
+					if( !fadeOut || fadeOut < 0.020 ) {
+						sound.audio.pause();
+					} else {
+						var volume = sound.audio.volume,
+							orgVolume = volume;
```

Comment:
I assumed that you were setting `orgVolume = volume` so that it can be reset at a later time when `fadeOutStopInterval` is called.  The problem I see with this code is that if `stopSound` is called during a fade out `orgVolume` could be less than one might expect.  Sorry, I'm probably missing something obvious, but I don't see where the interval is cancel when calling `stopSound()`.